lib/react.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const { plugin } = require('typescript-eslint')
+// @ts-ignore
+const reactPlugin = require('eslint-plugin-react')
+
+/**
+ * @typedef { 'recommended' | 'all' } ReactConfig
+ */
+
+/**
+ * @typedef ReactifyOptions
+ * @property {string[]|undefined} [ignores]
+ * @property {string} [name]
+ * @property {import('eslint').Linter.FlatConfig|undefined} tsConfig
+ * @property {ReactConfig|undefined} [reactConfig]
+ * @property {boolean} [requireReactImport]
+ */
+
+/**
+ * @param {ReactifyOptions} [options]
+ * @returns {Array<import('eslint').Linter.FlatConfig>}
+ */
+function reactify (options) {
```

Comment:
The function signature should be similar to typescriptify one: (previousConfig, reactConfig): mergedConfig.